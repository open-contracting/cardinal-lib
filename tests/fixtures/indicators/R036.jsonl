{"ocid":"F","bids":{"details":[{"status":"disqualified","value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active"}]}
{"ocid":"invited","bids":{"details":[{"status":"disqualified","value":{"amount":10,"currency":"USD"}},{"status":"invited","value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active"}]}
{"ocid":"withdrawn","bids":{"details":[{"status":"disqualified","value":{"amount":10,"currency":"USD"}},{"status":"withdrawn","value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active"}]}
{"ocid":"pending","bids":{"details":[{"status":"disqualified","value":{"amount":10,"currency":"USD"}},{"status":"pending","value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active"}]}
{"ocid":"valid","bids":{"details":[{"status":"disqualified","value":{"amount":10,"currency":"USD"}},{"status":"valid","value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active"}]}
{"ocid":"equal","bids":{"details":[{"status":"disqualified","value":{"amount":1,"currency":"USD"}},{"status":"valid","value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active"}]}
{"ocid":"nonfinal","bids":{"details":[{"status":"disqualified","value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"pending"}]}
{"ocid":"cancelled tender","bids":{"details":[{"status":"disqualified","value":{"amount":1,"currency":"USD"}}]},"awards":[{"status":"active"}],"tender":{"status":"cancelled"}}
