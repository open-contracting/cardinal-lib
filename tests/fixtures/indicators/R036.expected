{"OCID":{"1":{"R036":1.0},"invited":{"R036":1.0},"withdrawn":{"R036":1.0}}}
